src/character.h

```diff
@@ -484,6 +484,7 @@ class Character : public Creature, public visitable<Character>
         std::vector<bionic> my_bionics;
 
     protected:
+        virtual void on_stat_change( const std::string &, int ) override {};
```

Comment:
I'd appreciate an answer for my last question. What is the most appropriate place for the stats enum?